plugin/src/Assets.lua

```diff
@@ -45,6 +45,9 @@ local Assets = {
 			[500] = "rbxassetid://2609138523"
 		},
 	},
+	Sounds = {
+		Notification = "rbxassetid://9119690001",
```

Comment:
Consider replacing with a [much nicer sound](https://www.roblox.com/library/255881176); that imo, makes more sense UX-wise!
```suggestion
		Notification = "rbxassetid://255881176",
```

Reply:
It has to be something we have the right to use, not someone reuploading a Windows sound.

Reply:
Ahhh, far point!